chaosreliably/controls/experiment.py

```diff
@@ -0,0 +1,310 @@
+from typing import Any, cast
+
+from chaoslib.types import Configuration, Experiment, Secrets
+from logzero import logger
+
+from chaosreliably import get_session
+from chaosreliably.types import (
+    EntityContext,
+    EntityContextExperimentEventLabels,
+    EntityContextExperimentLabels,
+    EntityContextExperimentRunLabels,
+    EntityContextExperimentVersionLabels,
+    EntityContextMetadata,
+    EventType,
+)
+
+__all__ = ["before_experiment_control"]
+
+
+def before_experiment_control(
+    context: Experiment,
+    configuration: Configuration = None,
+    secrets: Secrets = None,
+    **kwargs: Any,
+) -> None:
+    """
+    Control run *before* the execution of an Experiment.
+
+    For a given Experiment, the control creates (if not already created) an Experiment
+    Entity Context and an Experiment Version Entity Context in the Reliably Entity
+    Server.
```

Comment:
No need to specify the name of the service. `Reliably Entity Server` should be `Reliably services`. The ES is not a public thing.

Reply:
Good shout, done